lamp/tasks/install_php.yml

```diff
@@ -16,7 +16,7 @@
   apt:
       name: "php{{ '' if php_version == '7.0' else php_version }}-{{ item }}"
       state: present
-  with_items: "{{ ['fpm', 'cli', 'mysql'] + php_modules | default([]) }}"
+  with_items: "{{ ['fpm', 'cli', 'mysql', 'zip'] + php_modules | default([]) }}"
```

Comment:
This appears to be specific to PHP 5.6 — zip functionality is included automatically for 5.5 and 7. Can we move this to the vars file for the Cov. Life project, perhaps with a note in the development-standard repo to include zip in the modules array if using 5.6?